action.yml

```
@@ -0,0 +1,50 @@
+name: 'Current projects'
+description: |
+  Github Action to determine all projects a pull request has cards on.
+
+  Returns JSON like this:
+
+    [
+      {
+        "databaseId": 123,
+        "name": "my-github-project"
+      }
+    ]
+
+outputs:
+  current-projects:
+    description: "Projects a pull request has cards on"
+    value: ${{ steps.get-current-projects.outputs.current-projects }}
+runs:
+  using: composite
+
+
+  steps:
+    - id: get-current-projects
```

The number of newlines looks non-uniform. Maybe:

```suggestion
outputs:
  current-projects:
    description: "Projects a pull request has cards on"
    value: ${{ steps.get-current-projects.outputs.current-projects }}

runs:
  using: composite

  steps:
    - id: get-current-projects
```